bgpd/bgp_main.c

```diff
@@ -407,6 +415,9 @@ int main(int argc, char **argv)
 	/* BGP related initialization.  */
 	bgp_init();
 
+	zclient->instance = instance;
+	zclient->redist_default = ZEBRA_ROUTE_BGP;
```

Comment:
This makes no sense to me.  This is already done when bgp calls zclient_init.  Why is this needed?

Reply:
Thank you, I'll correct it.